docs/hvordan-utvikle-maler.adoc

```diff
@@ -0,0 +1,550 @@
+:docinfo: shared
+:source-highlighter: highlight.js
+:toc:
+
+= Utvikling av brevmaler i brevbakeren
+
+Brevbakeren inneholder brevmaler i form av vår egen Kotlin DSL(Domain specific language)
+DSL'en er utviklet for å kun inneholde de elementene vi trenger for å representere innholdet i brev i nav.
+
+Vi har med vilje utelatt enkelt funksjonalitet som ganging og deling ettersom vi mener at forretningslogikk bør holdes utenfor brevmaler.
```

Comment:
forslag: Vi har bevisst utelatt funksjonalitet som ikke direkte har med visningslogikk å gjøre, f. eks aritmetiske operasjoner for tall.

Reply:
Blir for profft :laughing: Ifix.